### ISSUE TITLE
Button Text Overlapping Symbol Issue (Mobile Version)

### ISSUE THREAD

**Issue by Pritish-Priyam** [processing/p5.js-website#450]
### Topic

Hi there,

I've noticed a minor flaw in the UI while I was accessing the website on my mobile. On the Accessibility button, the text is overlapping the symbol, making it hard to read. Here are the details:

**Issue:** Text on the button overlaps with the symbol.
**Location:** Landing Page
**Screenshot:** 
![Accessibility_p5js](https://github.com/processing/p5.js-website/assets/91785302/43dd7397-44e9-497c-bd30-749233560f4a)

**Steps to Reproduce:**

Go to the Home Page
Locate the button with the overlapping issue
Observe the text and symbol on the button

**Suggestions:**
Increase the padding or margin between the text and the symbol.

Additionally, I would love to work on this issue and contribute to improving the platform. Please let me know if I can assist in resolving this.

**Comment by Qianqianye** [comment]
Thanks @Pritish-Priyam.
It seems like a design issue for small mobile devices. I'm tagging the website designers @stefanietam @tyler-yin to offer some insights.

**Comment by tyler-yin** [comment]
Hello! Thank you for flagging this visual bug. Based on the designs, there should be a smaller gap between the language and accessibility dropdown menu/button in our design. The column-gap should be 20px (not 40px) for mobile widths.

See screenshot below for reference:

<img width="396" alt="Screenshot 2024-07-20 at 11 03 52 AM" src="https://github.com/user-attachments/assets/0f9ebfc5-a32f-4c8b-846a-6beaf1f19919">

**Comment by Pritish-Priyam** [comment]
@tyler-yin 40px seems to work quite effectively when it comes to major mobile devices, but you'll observe that on a width of 375px on devices like the iPhone SE it breaks down.  I believe this can be reproduced on various Fold devices as well.

**Comment by Pritish-Priyam** [comment]
Is it worth adding a breakpoint for the same in the global CSS file?

**Comment by eebadreza** [comment]
Hello is this issue still open...

**Comment by aakansha2304** [comment]
Can I work on this issue?

**Comment by limzykenneth** [comment]
Sorry for the late reply. @eebadreza If you are still interested, I'll assign the issue to you, otherwise I can assign @aakansha2304 to work on this. Thanks.

**Comment by Pratik5252** [comment]
Is this issue still open and can I work on this issue?
